### PR TITLE
Plotting updates

### PR DESCRIPTION
**Pull request by varunagrawal**
- Fixed plotting functions
- Renamed `P` to more descriptive `covariance`
- Added docstrings

---
This change is [Reviewable](https://reviewable.io/reviews/borglab/gtsam/341)

### CODE REVIEW

File: cython/gtsam/utils/plot.py [hunk 1]
```diff
@@ -44,16 +59,24 @@ def ellipsoid(xc, yc, zc, rx, ry, rz, n):
     return x, y, z
 
 
-def plot_covariance_ellipse_3d(axes, origin, P, scale=1, n=8, alpha=0.5):
+def plot_covariance_ellipse_3d(axes, origin, covariance, scale=1, n=8, alpha=0.5):
```
Comment (dellaert) [inline cython/gtsam/utils/plot.py:62]: API change?

File: cython/gtsam/utils/plot.py [hunk 2]
```diff
@@ -105,33 +136,64 @@ def plot_pose2_on_axes(axes, pose, axis_length=0.1, covariance=None):
...
     fig = plt.figure(fignum)
     axes = fig.gca()
-    plot_pose2_on_axes(axes, pose, axis_length, covariance)
+    plot_pose2_on_axes(axes, pose, axis_length=axis_length,
+                       covariance=covariance)
 
 
-def plot_point3_on_axes(axes, point, linespec, P=None):
-    """Plot a 3D point on given axis 'axes' with given 'linespec'."""
+def plot_point3_on_axes(axes, point, linespec, covariance=None):
```
Comment (dellaert) [inline cython/gtsam/utils/plot.py:155]: API change?

**Comment by varunagrawal** [comment]
Uhhhh I thought about that as soon as I hit enter. We really should do a version bump soon. :sweat:

**Comment by varunagrawal** [comment]
Undid the changes.

**Pull request merged**